rules/rules_test.go

```diff
@@ -498,6 +498,114 @@ func TestExpressions(t *testing.T) {
 			intervalRanges: 2,
 		},
 		{
+			expr: `floor(0.004 * http_requests{group="production",job="api-server"})`,
+			output: []string{
+				`{group="production", instance="0", job="api-server"} => 0 @[%v]`,
+				`{group="production", instance="1", job="api-server"} => 0 @[%v]`,
+			},
+			fullRanges:     0,
+			intervalRanges: 2,
+		},
+		{
+			expr: `ceil(0.004 * http_requests{group="production",job="api-server"})`,
+			output: []string{
+				`{group="production", instance="0", job="api-server"} => 1 @[%v]`,
+				`{group="production", instance="1", job="api-server"} => 1 @[%v]`,
+			},
+			fullRanges:     0,
+			intervalRanges: 2,
+		},
+		{
+			expr: `round(0.004 * http_requests{group="production",job="api-server"})`,
+			output: []string{
+				`{group="production", instance="0", job="api-server"} => 0 @[%v]`,
+				`{group="production", instance="1", job="api-server"} => 1 @[%v]`,
+			},
+			fullRanges:     0,
+			intervalRanges: 2,
+		},
+		{ // Round should correctly handle negative numbers
+			expr: `round(-1 * (0.004 * http_requests{group="production",job="api-server"}))`,
+			output: []string{
+				`{group="production", instance="0", job="api-server"} => -0 @[%v]`,
+				`{group="production", instance="1", job="api-server"} => -1 @[%v]`,
+			},
+			fullRanges:     0,
+			intervalRanges: 2,
+		},
+		{ // Round should round halft to even
```

Comment:
Hmm. What's "halft"? Half?

Periods at the end of the comments here and around please.